API_CHANGES.md

```diff
@@ -10,11 +10,11 @@ Add in the linux `task.get_threads` method added to the API.
 
 2.0.3
 =====
-`DEVICE_OBJECT.get_attached_devices` and `DRIVER_OBJECT.get_devices` added to the API.
+Add in the windows `DEVICE_OBJECT.get_attached_devices` and `DRIVER_OBJECT.get_devices` method added to the API.
```

Comment:
Technically there's two items mentioned, so the number of `method` should be plural (`methods`).  Also "Add in the... added to the API" doesn't make much sense.  Happy to make it clearer, but best we change properly so there isn't another follow-up pull request.  5;)

I'd go with:

Add in the windows  `DEVICE_OBJECT.get_attached_devices` and `DRIVER_OBJECT.get_devices` methods to the API.

Reply:
All right, I'll change the sentence below the same.